packages/storage/observable.test.ts

```diff
@@ -121,23 +127,15 @@ describe("ObservableStorage", () => {
       async (state) => {
         s2 = state
       },
-      ["a", "b"]
-    )
-
-    // Not matched
-    let s3: ReturnType<(typeof s)["get"]>
-    s.subscribe(
-      async (state) => {
-        s3 = state
-      },
-      ["a", "b", "d"]
+      {
+        a: { b: {} }
+      }
     )
 
     s.set({ a: { b: { c: 999 } } })
 
     expect(s1.a.b.c).toBe(999)
     expect(s2.a.b.c).toBe(999)
-    expect(s3).toBeUndefined()
   })
 
```

Comment:
Should add the "no match" test to make the test more comprehensive:

```typescript
  it("Should be able to prevent subscribing to no match path", () => {
    const s = new ObservableStorage({ a: { b: { c: 123, d: 456 } } })

    // No match
    let stateSubscribed: ReturnType<(typeof s)["get"]>
    s.subscribe(
      async (state) => {
        stateSubscribed = state
      },
      {
        a: {
          b: {
            d: 0
          }
        }
      }
    )

    s.set({ a: { b: { c: 999 } } })

    expect(stateSubscribed).toBeUndefined()
  })
```